samples/monster_generated.h

```diff
@@ -25,6 +25,14 @@ enum Color {
   Color_MAX = Color_Blue
 };
 
+inline Color (&EnumValuesColor())[3] {
```

Comment:
My first implementation was std::vector, but this generated template instantiations for each enumeration class when using the scoped_enums parser option. A debug binary increased in size by 40KB for 15 enumeration types when comparing std::vector to an array refererence.